policies/tagging/README.md

```diff
@@ -12,9 +12,6 @@ All environments and resources:
 ## Allowed values
 Tags and their allowed values.
 
-application:
```

Comment:
this should still be included, required, an approved application name: see https://github.com/hmcts/terraform-module-common-tags/blob/master/team-config.yml#L13